spec/lib/jobs_spec.rb

```diff
@@ -21,8 +26,16 @@
     end
   end
 
+  describe 'job_scrub(query)' do
```

Comment:
The Rspec standard is to preface class methods in a `describe` block with a dot, or a hash for instance methods:
```ruby
describe '.job_scrub(query)' do
```
https://github.com/rubocop-hq/rspec-style-guide#example-group-naming